xml/System.Net/Dns.xml

```diff
@@ -708,7 +708,9 @@
  If an empty string is passed as the `hostNameOrAddress` argument, then this method returns the IPv4 and IPv6 addresses of the local host.
   
  IPv6 addresses are filtered from the results of the <xref:System.Net.Dns.GetHostAddresses%2A> method if the local computer does not have IPv6 installed. As a result, it is possible to get back an empty <xref:System.Net.IPAddress> instance if only IPv6 results were available for the `hostNameOrAddress` parameter.  
-  
+ 
+ This method follows standard hostname resolution rules on Windows, so if a host is described in the `hosts` file, the IP address(es) there will be returned instead of querying DNS.
```

Comment:
I know you're probably going to make changes to your PR, but don't (es) to plurals since this is hard to translate.

Source: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/s-es